lib/rack/tracker/version.rb

```diff
@@ -1,5 +1,5 @@
 module Rack
   class Tracker
-    VERSION = '1.3.1'
+    VERSION = '1.3.3'
```

Comment:
while I really appreciate your engagement, 
please let the decision _when_ and _how_ to pump version to the maintainers. ;)